src/openmc_nek_driver.cpp

```diff
@@ -165,15 +166,20 @@ void OpenmcNekDriver::init_mappings()
 
 void OpenmcNekDriver::init_tallies()
 {
+  using gsl::index;
+  using gsl::narrow_cast;
+  using openmc::CellInstance;
+
   comm_.message("Initializing tallies");
 
   if (openmc_driver_->active()) {
     // Build vector of material indices
-    std::vector<int32_t> mats;
+    std::vector<CellInstance> instances;
```

Comment:
I know the `using` statement isn't very far away, but could you change this declaration to `std::vector<openmc::CellInstance>`?  As long as we have `CellInstance` in two namespaces, I'd like the declarations to be as explicit as possible